server/auth.py

```diff
@@ -109,6 +134,12 @@ def get_responses_dict(userEmail : str, db : firestore.firestore.Client):
     responses_ref = db.collection("responses")
     return responses_ref.document(email_hash(userEmail)).get().to_dict()
 
+def get_predictions_dict(userEmail : str, db : firestore.firestore.Client):
+    predictions_ref = db.collection("prediction_users").document(userEmail).collection("predictions")
+    docs = predictions_ref.stream()
+    docs_dict = {doc.id:int(doc.to_dict()["guess"]) for doc in docs}
```

Comment:
Can we fix the style here please. Add some spacing because it's really hard to tell what's going on otherwise.